config/routes.rb

```diff
@@ -4,6 +4,8 @@
   match '422' => 'errors#unprocessable_entity', via: :all
   match '500' => 'errors#internal_server_error', via: :all
 
+  get 'up' => 'rails/health#show', as: :rails_health_check
```

Comment:
New default feature / route, don't think Heroku uses it but doesn't hurt to leave it for future compatibility.

Truly a beautiful page

<img width="318" height="408" alt="Image" src="https://github.com/user-attachments/assets/80e3c964-9c4d-46a1-8dc3-5ec5ce3cf1dd" />